src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -208,12 +215,19 @@ object EasyIngest {
       System.exit(13)
     }
 
+  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
+  private class CompositeExceptionMessages(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)}"))
   private implicit class ListTryExtensions[T](xs: List[Try[T]]) {
     def sequence: Try[List[T]] =
       if (xs.exists(_.isFailure))
         Failure(new CompositeException(xs.collect { case Failure(e) => e }))
       else
         Success(xs.map(_.get))
-  }
 
+    def squashedSequence: Try[List[T]] =
```

Comment:
Only the pull request slipped through the filter to my private mailbox. The pull request looked good to me but I did not see this discussion in time. Sorry.